README.md

```diff
@@ -1 +1 @@
-# Review-of-my-project-from-Trials
\ No newline at end of file
+# LandingPageTrials
```

Comment:
Good job 😃 so far!
You still need to work on some issues to go to the next project, but you are almost there!

- [ ] Kindly edit your README.md file. Your Readme file does not contain your project details. This is very important because it allows you to describe your project by browsing all the tools used to carry out this project. you can use the [Microverse Readme template](https://github.com/microverseinc/readme-template) to make it easier for you.